README.md

```diff
@@ -1,11 +1,14 @@
 # @brightcove/react-player-loader
-A React component to load a brightcove player in the browser.
+A React component to load a Brightcove Player in the browser.
+
+## Brightcove Player Support
+Currently, this library supports Brightcove Players v6.11.0 and higher.
 
 ## Table of Contents
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-## Installation
+- [Installation](#installation)
```

Comment:
installation is listed twice now for some reason, hmmm

Reply:
doctoc bein’ weird